apps/web/src/app/[locale]/challenge/_components/description/index.tsx

```diff
@@ -43,6 +44,8 @@ export function Description({ challenge }: Props) {
   const [hasBookmarked, setHasBookmarked] = useState(challenge.bookmark.length > 0);
   const session = useSession();
 
+  const isAotChallenge = useMemo(() => AOT_CHALLENGES.includes(challenge.slug), [challenge.slug]);
```

Comment:
lgtm. another way to do this might have been checking if `challenge.track.slug === 'aot-2023'`, but this accomplishes the same